### PR TITLE
Update E2E versions to 1.5.0

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by berndverst — APPROVED** [review]

File: tests/e2e/standalone/standalone_test.go [hunk 1]
```diff
@@ -243,12 +245,15 @@ func testInstall(t *testing.T) {
...
+			output, err := spawn.Command(file, "--version", "2>/dev/null")
 			if !assert.NoError(t, err) {
 				return
 			}
 			output = strings.TrimSpace(output)
-			if !assert.Equal(t, version, output) {
+			// Changing version check since there is a log that is output on daprd --version
+			// 2021/11/12 11:10:38 maxprocs: Leaving GOMAXPROCS=12: CPU quota undefined
+			// before the version is output
+			if !assert.Contains(t, output, version) {
```
Comment (wcs1only) [inline tests/e2e/standalone/standalone_test.go:256]: Do we need both "2>/dev/null" above and assert.Contains? Should only need one or the other?

Comment (berndverst) [reply tests/e2e/standalone/standalone_test.go:256]:
```suggestion
			if !assert.Equal(t, output, version) {
```

File: tests/e2e/standalone/standalone_test.go [hunk 2]
```diff
@@ -243,12 +245,15 @@ func testInstall(t *testing.T) {
...
+			output, err := spawn.Command(file, "--version", "2>/dev/null")
 			if !assert.NoError(t, err) {
 				return
 			}
 			output = strings.TrimSpace(output)
-			if !assert.Equal(t, version, output) {
+			// Changing version check since there is a log that is output on daprd --version
+			// 2021/11/12 11:10:38 maxprocs: Leaving GOMAXPROCS=12: CPU quota undefined
+			// before the version is output
+			if !assert.Equal(t, output, version) {
```
Comment (berndverst) [inline tests/e2e/standalone/standalone_test.go:256]:
```suggestion
			if !assert.Equal(t, version, output) {
```

**Review by berndverst — DISMISSED** [review]

File: tests/e2e/standalone/standalone_test.go [hunk 3]
```diff
@@ -243,11 +245,14 @@ func testInstall(t *testing.T) {
...
-			output, err := spawn.Command(file, "--version")
+			output, err := spawn.Command(file, "--version", "2>/dev/null")
 			if !assert.NoError(t, err) {
 				return
 			}
 			output = strings.TrimSpace(output)
+			// Changing version check since there is a log that is output on daprd --version
+			// 2021/11/12 11:10:38 maxprocs: Leaving GOMAXPROCS=12: CPU quota undefined
+			// before the version is output
 			if !assert.Equal(t, version, output) {
```
Comment (berndverst) [inline tests/e2e/standalone/standalone_test.go:256]:
```suggestion
			if !assert.Contains(t, output, version) {
```

File: tests/e2e/standalone/standalone_test.go [hunk 4]
```diff
@@ -243,11 +245,14 @@ func testInstall(t *testing.T) {
 				return
 			}
 			// Check version
-			output, err := spawn.Command(file, "--version")
+			output, err := spawn.Command(file, "--version", "2>/dev/null")
```
Comment (berndverst) [inline tests/e2e/standalone/standalone_test.go:248]:
```suggestion
			output, err := spawn.Command(file, "--version")
```

**Review by berndverst — APPROVED** [review]

**Pull request merged**